parastell/magnet_coils.py

```diff
@@ -291,6 +313,10 @@ def export_mesh(self, mesh_filename="magnet_mesh", export_dir=""):
             filename=mesh_filename, export_dir=export_dir
         )
 
+        # Delete magnet mesh to prevent inclusion in future Cubit mesh exports
+        volume_ids_str = " ".join(str(id) for id in self.volume_ids)
+        cubit.cmd(f"delete mesh volume {volume_ids_str} propagate")
```

Comment:
While not useful now, it will be when we incorporate in-vessel component meshing (coming up soon)